scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,14 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$TempPath = Join-Path -Path $env:TEMP -ChildPath "getclrdbg"
```

Comment:
I don't think this is the right way to go, but you make a good point about two things trying to run this script simultaneously. 

---
In reply to: [63795430](https://github.com/Microsoft/MIEngine/pull/332#discussion_r63795430) [](ancestors = 63795430)